packages/aws-cdk-lib/aws-ecs-patterns/lib/ecs/network-multiple-target-groups-ecs-service.ts

```diff
@@ -148,9 +148,15 @@ export class NetworkMultipleTargetGroupsEc2Service extends NetworkMultipleTarget
       this.addPortMappingForTargets(this.taskDefinition.defaultContainer, props.targetGroups);
       this.targetGroup = this.registerECSTargets(this.service, this.taskDefinition.defaultContainer, props.targetGroups);
     } else {
+      const containerPort = this.taskDefinition.defaultContainer.portMappings[0].containerPort;
+
+      if (!containerPort) {
+        throw new Error('You must specify a containerPort');
```

Comment:
No real reason, I think at some point during development I rewrote the message in one place but not the other. The text below seems a little clearer on what the expectations are, so I'd stick with it

Reply:
No real reason, I think at some point during development I rewrote the message in one place but not the other. The text below seems a little clearer on what the expectations are, but with some adjustments it would be even better I think. What do you think about this version?

>The first port mapping added to the default container must expose a single port

Reply:
Perfect!